pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/certmanager/kustomizeconfig.go

```diff
@@ -44,20 +44,12 @@ func (f *KustomizeConfig) SetTemplateDefaults() error {
 }
 
 //nolint:lll
-const kustomizeConfigTemplate = `# This configuration is for teaching kustomize how to update name ref and var substitution 
+const kustomizeConfigTemplate = `# This configuration is for teaching kustomize how to update name ref substitution 
 nameReference:
 - kind: Issuer
   group: cert-manager.io
   fieldSpecs:
   - kind: Certificate
     group: cert-manager.io
     path: spec/issuerRef/name
-
```

Comment:
@laxmikantbpandhare so we no longer need these ones? Seems like worked and passed successfully on the tests

Reply:
@camilamacedo86 - https://kubernetes.slack.com/archives/C9A5ALABG/p1657034450354619?thread_ts=1656957920.477289&cid=C9A5ALABG We have s comment from someone on `kustomize` for the same question.

Reply:
we could check that is all fine now. thank you